### DISCUSSION TITLE
Update Content

### DISCUSSION THREAD

**Discussion by Galaxy-VN** [NichtStudioCode/InvUI#94] [Q&A] [answered]
Okay so i am doing a lobby for game and i have a list of game in it and item represent for game so what i want to ask is when the some game is remove in list i have to update it again but the content for PagedGui but i have no idea how to expect i have to reopen gui so that content will be load again before set in

**Comment by rexlManu** [comment, marked as answer]
What I'm doing to handle such a situation is that I have a "Context" class with Player, PagedGUI and other possible values and when the contents are changed like in my example filtering, I'm accessing the built PagedGUI via the context in my click listener like SimpleItem for example and call setContent.

**Reply by Galaxy-VN** [reply]
thanks!
